index.html

```diff
@@ -95,5 +95,12 @@
     <script type="text/javascript">
   $('select').select2();
 </script>
+  <script> 
+    $(document).ready(function(){
+     
+       $("option[disabled='disabled']").append('(Assigned)');
```

Comment:
Can we add a "non-breaking space" between the end of ", NY" and the start of the (Assigned) 

Try a Google search for "html non-breaking space"